res/controllers/Reloop Terminal Mix 2-4.js

```diff
@@ -121,6 +121,11 @@ TerminalMix.samplerVolume = function (channel, control, value) {
     }
 }
 
+TerminalMix.pitchSlider = function (channel, control, value, status, group) {
+	// invert pitch slider (down=faster) so it matches the labels on controller
+	engine.setValue(group,"rate",-script.midiPitch(control, value, status));
+}
+
```

Comment:
You're not using script.midiPitch() correctly. It takes the LSB and the MSB as the first two arguments, so you have to have two separate functions, one that is mapped to the MSB and one that reacts to the LSB. Store the MSB and call script.midiPitch when the LSB changes (correct me if I have that backwards).